docs/_includes/generated/yarn_config_configuration.html

```diff
@@ -22,6 +22,11 @@
             <td style="word-wrap: break-word;">"0"</td>
             <td>With this configuration option, users can specify a port, a range of ports or a list of ports for the Application Master (and JobManager) RPC port. By default we recommend using the default value (0) to let the operating system choose an appropriate port. In particular when multiple AMs are running on the same physical host, fixed port assignments prevent the AM from starting. For example when running Flink on YARN on an environment with a restrictive firewall, this option allows specifying a range of allowed ports.</td>
         </tr>
+        <tr>
```

Comment:
you might also need to change `yarn-setup.md` to add some examples into the launch CLI script.

Reply:
ok, I have added the instruction in yarn-setup.md and yarn-setup.zh.md(used in English).

Reply:
could you also add information indicate that this only works for newer version of YARN like we discussed in the JIRA ticket? based on what I researched this is only enabled on YARN 2.8+, not sure if it was backported to any older version.

Reply:
yes, I have added the supported version info in the config description.

Reply:
@wzhero1 we might want to revert the 2.8.x constrain here. and say something like this.
```
Please refer to YARN official documentation for specific settings required to enable priority schedule for the targeted YARN version
```

sorry for the confusion :-(

Reply:
@walterddr I also found in 2.8.x,  the `ApplicationReport` which is generated by `yarnClient.getApplicationReport` has the method `set/getPriority`, but there is no such method in 2.6.x. 

> Hi here,
> 
> did a bit of research: the API is already there in: [ApplicationSubmissionContext](https://hadoop.apache.org/docs/r2.4.1/api/org/apache/hadoop/yarn/api/records/ApplicationSubmissionContext.html). However, the usage pattern is different: it is only introduced to Capacity scheduler in [2.8.x](https://hadoop.apache.org/docs/r2.8.5/hadoop-yarn/hadoop-yarn-site/CapacityScheduler.html#Setup_for_application_priority).
> 
> The priority setting was used in [ResourceRequest](http://hadoop.apache.org/docs/r2.4.1/hadoop-yarn/hadoop-yarn-site/WritingYarnApplications.html), which is a different usage pattern: after [2.6.x](https://hadoop.apache.org/docs/r2.6.5//hadoop-yarn/hadoop-yarn-site/WritingYarnApplications.html), YARN documentation indicates that `appContext.setPriority(...)` should be used instead of `rsrcRequest.setPriority(...)`
> 
> I am still not sure how prior to 2.6.x what is the effect of setting `appContext.setPriority`.